src/event_handler.rs

```diff
@@ -90,7 +90,7 @@ pub async fn events_listener(app_state: Data<AppState>, starting_block: U64) {
         let jobs_event_filter = Filter::new()
             .address(app_state.jobs_contract_addr)
             .topic0(vec![
-                keccak256("JobCreated(uint256,address,bytes32,bytes,uint256,address[])"),
+                keccak256("JobCreated(uint256,uint8,address,bytes32,bytes,uint256,address[])"),
```

Comment:
environment is indexed field. You can use it in filter?

Reply:
This stream is capturing all the relevant events of `Jobs` contract i.e., `JobCreated` and `JobResponded`, so when I added the env filter to it, the `JobResponded` events got affected. That's why have added the check later when an `JobCreated` event is received.

Reply:
Hmm, that's tricky. Just check if there is another option we can use, this will just avoid handling of unnecessary events.